Bottom: a19cb475f20c185601fe8b7f9c689ae41376b75a
Top:    84d52bfd4755ac83a265343221ffa0c7023a8bf8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 23:55:36 +0100

misc.c: Remove comment missed from commit e82ad9b


---

diff --git a/src/misc.c b/src/misc.c
index befeeed..3dc87c6 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -764,9 +764,6 @@ GetDataBoundingBox (DataTypePtr Data)
 
 /* ---------------------------------------------------------------------------
  * centers the displayed PCB around the specified point (X,Y)
- * if Delta is false, X,Y are in absolute PCB coordinates
- * if Delta is true, simply move the center by an amount X, Y in screen
- * coordinates
  */
 void
 CenterDisplay (LocationType X, LocationType Y)
